src/framework/accessibility/internal/accessibilitycontroller.h

```diff
@@ -49,10 +49,11 @@ namespace mu::accessibility {
 class AccessibilityController : public IAccessibilityController, public IAccessible, public async::Asyncable,
     public std::enable_shared_from_this<AccessibilityController>
 {
-    INJECT(framework::IApplication, application)
-    INJECT(ui::IMainWindow, mainWindow)
-    INJECT(ui::IInteractiveProvider, interactiveProvider)
-    INJECT(IAccessibilityConfiguration, configuration)
+public:
```

Comment:
So, we should add public if we are writing tests for this class

Reply:
Unfortunately, yes, or a friendly class